distribution/changelog.txt

```diff
@@ -1,18 +1,24 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: OpenGL renderengine added.
 - Feature: Ability to rotate map elements with the tile inspector.
 - Feature: Add cheat to disable littering.
 - Feature: Add ride console command for diagnostics and changing vehicle type.
 - Feature: Allow setting ownership of map edges.
 - Feature: Allow up to 255 cars per train.
 - Feature: Importing SV4 and SC4 files with rides.
-- Feature: Added Norwegian translation
+- Feature: Add Norwegian translation and currency.
```

Comment:
> and currency.

The currency was never added, it was a change of the existing currency in game
You should remove that as it's the swedish currency you're referring